### PR TITLE
fix MRO

### PR DESCRIPTION
**Pull request by jensens**
for some reason the MRO is completly broken in plone.dexterity, this is a try to fix it.

### CODE REVIEW

**Comment by jensens** [comment]
test here http://jenkins.plone.org/job/pull-request-5.0/75/ .oO( curious )

**Comment by jensens** [comment]
further reading about pythons MRO http://makina-corpus.com/blog/metier/2014/python-tutorial-understanding-python-mro-class-search-path

File: plone/dexterity/content.py [hunk 1]
```diff
@@ -634,20 +635,20 @@ class Item(PasteBehaviourMixin, BrowserDefaultMixin, DexterityContent):
...
 class Container(
-        PasteBehaviourMixin, DAVCollectionMixin, BrowserDefaultMixin,
-        CMFCatalogAware, CMFOrderedBTreeFolderBase, DexterityContent):
+    DexterityContent,
+    PasteBehaviourMixin,
+    DAVCollectionMixin,
+    BrowserDefaultMixin,
+    CMFCatalogAware,
+    CMFOrderedBTreeFolderBase,
+    DexterityContent
```
Comment (jone) [inline plone/dexterity/content.py]: ``DexterityContent`` is duplicate (first and last superclass) - is this by intention or just a copy/paste error?

Comment (jensens) [inline plone/dexterity/content.py]: ops - its by mistake

**Comment by jensens** [comment]
next trial http://jenkins.plone.org/job/pull-request-5.0/96/

**Comment by jensens** [comment]
http://jenkins.plone.org/job/pull-request-5.0/97/

**Comment by jensens** [comment]
quite scary overall

**Comment by jensens** [comment]
http://jenkins.plone.org/job/pull-request-5.0/110/

**Comment by jensens** [comment]
http://jenkins.plone.org/job/pull-request-5.0/166/

**Comment by jensens** [comment]
Note to future-self: 
at 
https://github.com/plone/plone.app.content/blob/master/plone/app/content/browser/actions.py#L149
the setDefaultPage is called before rename, but the event in 
https://github.com/zopefoundation/Zope/blob/master/src/OFS/CopySupport.py#L392
with its subscriber registered at
https://github.com/plone/Products.CMFDynamicViewFTI/blob/master/Products/CMFDynamicViewFTI/configure.zcml#L35
and defined at 
https://github.com/plone/Products.CMFDynamicViewFTI/blob/master/Products/CMFDynamicViewFTI/browserdefault.py#L251
should take care of it.

Task: Figure out why this does not work. This will give probably a hint why all the tests are failing after fixing MRO.

**Comment by jensens** [comment]
try again http://jenkins.plone.org/job/pull-request-5.0/276/

**Comment by jensens** [comment]
@jones help appreciated, this is a huge one. at the moment i do not have much time, otherwise

**Comment by jensens** [comment]
this needs to be rethought. i close this here, since its too much at once.

**Pull request closed without merging**